utils/lambda-creator.js

```diff
@@ -0,0 +1,74 @@
+const AWS = require('aws-sdk');
+
+const { LAMBDA_FUNCTION_NAME } = require('./constants');
+
+// Set the parameters.
+
+exports.createLambda = async (
+    functionName,
+    description,
+    token,
+    bucketName,
+    accessKey,
+    secretKey,
+    iamRoleArn,
+    region,
+    listEnvVariables,
+) => {
+    let variables = {};
+    listEnvVariables.forEach((envVariable) => {
+        variables = { ...variables, ...envVariable };
+    });
+    const nameZip =
+        functionName.split(' ').length > 0
+            ? functionName.split(' ').join('-')
+            : functionName;
+    const params = {
+        Code: {
+            S3Bucket: bucketName, // BUCKET_NAME
+            S3Key: `${nameZip}.zip`, // ZIP_FILE_NAME
+        },
+        FunctionName: functionName || LAMBDA_FUNCTION_NAME,
+        Handler: 'index.handler',
+        Role: iamRoleArn, // IAM_ROLE_ARN; e.g., arn:aws:iam::650138640062:role/v3-lambda-tutorial-lambda-role
```

Comment:
Suggestion: to improve user experience can we create the lambda execution role using AWS sdk, instead of making the user manually create it and copy paste the arn?